pymatgen/io/qchem/inputs.py

```diff
@@ -195,6 +214,13 @@ def __str__(self):
         if self.geom_opt is not None:
             combined_list.append(self.geom_opt_template(self.geom_opt))
             combined_list.append("")
+        # svp section
+        if self.svp:
```

Comment:
I think we intentionally used `is not None` to avoid possible ambiguities with True/False in Python. Probably this is fine as well, though.